tools/cluster.py

```diff
@@ -39,6 +39,7 @@
 default_spot_price = '0.10'
 default_worker_instances = '1'
 default_master_instance_type = 'm3.xlarge'
+default_driver_heap_size = '25G'
```

Comment:
Change the default to 12G or change the default master instance type to m3.2xlarge